lib/udf_findhotel_utils.rb

```diff
@@ -10,22 +10,25 @@ class UdfFindhotelUtils
             if not room:
               return None
             else:
-              adults = 0
-              children = 0
-              rooms = room.split('|')
-              for r in rooms:
-                ch = r.split(':')
-                adults += int(ch[0])
-                if len(ch) > 1:
-                  children += len(ch[-1].split(','))
+              try:
```

Comment:
If you log exceptions like that you will be able to see them in one of Redshift's system tables:
http://docs.aws.amazon.com/redshift/latest/dg/udf-logging-messages.html